src/main/steps/applicant1/hub-page/progress-bar.njk

```diff
@@ -55,20 +55,20 @@
     label: {
       text: conditionalOrderGranted
     },
-    complete: progressionIndex > 11,
+    complete: progressionIndex > 12,
```

Comment:
these progression indices look a bit like magic numbers. I haven't looked at the code so not sure how they're used. but it seems dangerous to rely on an index in this way. is it not possible to use an ordered list

Reply:
https://github.com/hmcts/nfdiv-frontend/blob/master/src/main/steps/applicant1/hub-page/sole/content.ts#L160-#L175

In terms of how the page content shows up it could probably be changed, but I can't think of a better way of implementing it for the progress bar.

 @velimd any opinions?

Reply:
We can just use the state names. the idea was to use the index of states as the states model changed a few times at the time so we didnt have to rely on it.

Not sure what you mean by using a ordered list?

Reply:
what I meant by ordered list was just a sequence, like we have the sequence of the pages. so you just pick the next item in the sequence, if you don't want to mention the name of the state explicitly